.travis.yml

```diff
@@ -9,29 +8,81 @@ branches:
   - staging
   - trying
   - master
+  # release branch
+  - /^release\/[\d\.]+$/
+  # release tag
+  - /^v[\d\.]+$/
 
 addons:
-  apt:
-    packages:
-    - snapd
+  snaps:
+  -  lxd
+
+stages:
+- name: test
+  # don't test on pushes to master or release pull requests or tag events
+  # TODO: enable release pull request testing when we have more CI power
+  if: NOT (( branch = master AND type = push )
+             OR ( head_branch =~ ^release/[\d\.]+$ ))
+      AND tag IS blank
+- name: upload to ppa:mir-team
+  if: type = push
+      AND repo = MirServer/wlcs
+      AND ( branch = master
+            OR branch =~ ^release/[\d\.]+$
+            OR tag =~ ^v[\d\.]+$ )
 
 install:
-  - sudo snap install lxd
-  - sudo addgroup --system lxd
   - sudo adduser $USER lxd
-  - sudo snap start lxd
-  - for I in 1 2 3 4 5 6 7 8 9 10 ; do [ -S /var/snap/lxd/common/lxd/unix.socket ] || sleep 5 ; done
-  - sudo lxd init --auto
+  - sudo apt remove --assume-yes lxd
+  - sudo /snap/bin/lxd init --auto
```

Comment:
I thought the snap does this on its own? Any case, `snap run lxd` is more idiomatic, if `lxd` itself isn't on `$PATH`.